config/initializers/devise.rb

```diff
@@ -1,3 +1,4 @@
+require 'omniauth-google-oauth2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.